util/libcrypto.num

```diff
@@ -5425,3 +5425,4 @@ ASN1_item_d2i_ex                        5552	3_0_0	EXIST::FUNCTION:
 ASN1_TIME_print_ex                      5553	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_get0_provider                  5554	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_CTX_get0_provider              5555	3_0_0	EXIST::FUNCTION:
+ERR_clear_last_constant_time            5556	3_0_0	EXIST::FUNCTION:
```

Comment:
For the master branch, this looks wrong. I'm not sure if the scripts have been updated, but that should probably say 3_1_0, but maybe 3_0_1 if we intend to merge this to the stable branch.

Reply:
Yeah. Please remove that line and call `make update`, that will create a correct entry. Of course please first rebase the patch against current master.